setup.py

```diff
@@ -0,0 +1,7 @@
+from setuptools import setup, find_packages
```

Comment:
Adding this (minimal) Python package configuration file allows us to install the Python package. This makes it easier to actually import your code in `src/` in your tests. Note also that I have added an (empty) `__init__.py` to `src/` - this is what makes it a package and allows the `setup()` function to find it with `find_packages()`.